tests/sympc/store/crypto_primitive_provider_test.py

```diff
@@ -191,10 +186,16 @@ def test_primitive_logging_model(get_clients) -> None:
 @pytest.mark.parametrize(
     "ops",
     [
-        ["beaver_mul", {"a_shape": [1, 5], "b_shape": [1, 5]}],
-        ["beaver_matmul", {"a_shape": [1, 2880], "b_shape": [2880, 10]}],
-        ["beaver_conv2d", {"a_shape": [1, 1, 28, 28], "b_shape": [5, 1, 5, 5]}],
-        ["fss_comp", {}],
+        ["beaver_mul", {"a_shape": [1, 5], "b_shape": [1, 5], "nr_parties": 2}],
+        [
+            "beaver_matmul",
+            {"a_shape": [1, 2880], "b_shape": [2880, 10], "nr_parties": 2},
```

Comment:
Q: Are the number of parties required?

Reply:
yes, else we get this error:

`TypeError: _get_triples() missing 1 required positional argument: 'nr_parties'`

Reply:
Ahh..it is ok how it is now :smile:

Reply:
no problem :D